js/gratipay/team.js

```diff
@@ -192,6 +191,7 @@ Gratipay.team = (function() {
                     if(d.success) {
                         Gratipay.notification(d.success, 'success');
                     }
+                    debugger;
```

Comment:
Removed in 9287109e04b4545cf586ecccc5d2f59938ccc51e. I'll probably squash it with the commit that introduced it.